projects/kit/directives/button-group/button-group.style.less

```diff
@@ -18,6 +18,8 @@
 
     tui-elastic-container {
         inline-size: 100%;
+        transition: opacity calc(var(--tui-duration-moderate, 300ms) / 3)
```

Comment:
We should remove transition from opacity so that it does not lag when you drag with your finger, besides it breaks height transition.

Reply:
fixed